### PR TITLE
show icon with Modal

### PR DESCRIPTION
**Pull request by takanorip**
## Changes
Changed the icon code info to be displayed in a modal.

https://github.com/ubie-oss/ubie-vitals-website/assets/18516475/bb1b757e-c61a-41c9-9279-252fdbadab2d

### CODE REVIEW

File: src/components/react/CopyButton.module.css [hunk 1]
```diff
@@ -16,6 +17,11 @@
   background-color: transparent;
 }
 
+.button.secondary {
```
Comment (takanorip) [inline src/components/react/CopyButton.module.css:20]: add black style

File: src/components/react/CopyButton.tsx [hunk 2]
```diff
@@ -44,7 +46,12 @@ const ButtonCopy: FC<Props> = ({ text, className = '', label, invert, block }) =
             <CheckAIcon />
           </span>
         ) : label ? (
-          label
+          <>
+            <span className={styles.icon}>
+              <CopyIcon />
+            </span>
+            <span>{label}</span>
```
Comment (takanorip) [inline src/components/react/CopyButton.tsx:53]: Modified that icons are displayed on copy buttons with labels.

File: src/components/react/IconList/IconWrapper.tsx [hunk 3]
```diff
@@ -1,99 +1,57 @@
...
-              <Box pt="xxs" pb="xxs">
-                <Text type="note" size="lg" color="main" bold textAlign="center">
-                  copy
-                </Text>
-              </Box>
-              <CopyButton label="Ubie UI" text={toUbieUIStatement(name)} block />
-              <CopyButton label="Ubie Icons" text={toUbieIconsStatement(name)} block />
-            </div>
-          </div>
+      <button className={styles.wrapper} type="button" onClick={() => setOpen(true)}>
```
Comment (takanorip) [inline src/components/react/IconList/IconWrapper.tsx:33]: Change to button

File: src/components/react/SimpleCode.tsx
Comment (takanorip) [inline src/components/react/SimpleCode.tsx:1]: This is a non syntax-highlight code component.

**Review by 8845musign — APPROVED** [review]
Great work

**Pull request merged**